### ISSUE TITLE
`Vanilla` or `vanilla`

### ISSUE THREAD

**Issue by itscynxx** [R2Northstar/NorthstarWiki#263]
Different parts of the wiki refer to vanilla Titanfall 2 in different capitalization 

Some parts refer to it as [`Vanilla`](https://r2northstar.gitbook.io/r2northstar-wiki/installing-northstar/northstar-installers/viper-guide#vanilla-menu)
And others refer to it as [`vanilla`](https://r2northstar.gitbook.io/r2northstar-wiki/installing-northstar/northstar-installers/viper-guide#vanilla-menu)

It should be noted that I am part of the problem as well, I go from labeling it one way to another every so often

The reason is:
Titanfall 2 without mods is vanilla, however, when talking about Northstar vs vanilla Titanfall 2, generally it's just shortened to Northstar vs vanilla, where it sort of makes more sense to capitalize the vanilla, as it's referring to something as a whole, rather than an aspect of something (if that makes sense? Sort of writing this with a headache)

Simply put, was just curious about input of if everything should be changed to one way regardless of context

Again, I am a lot of the problem here, just realized it while updating the vanilla section and was thinking about making a pr for it, but decided to get some input first
